bridges/relays/utils/src/relay_loop.rs

```diff
@@ -270,7 +270,8 @@ fn create_metrics_registry(prefix: Option<String>) -> Registry {
 	match prefix {
 		Some(prefix) => {
 			assert!(!prefix.is_empty(), "Metrics prefix can not be empty");
-			Registry::new_custom(Some(prefix), None).expect("only fails if prefix is empty; prefix is not empty; qed")
+			Registry::new_custom(None, Some(iter::once((String::from("chain"), prefix)).collect()))
```

Comment:
I'm not too familar with Prometheus, so why is it better to use a `label` instead of a `prefix` here?